Admin/exercise_3_join_validator.md

```diff
@@ -11,6 +11,7 @@ Axelar Network is a work in progress. At no point in time should you transfer an
 
 ## Prerequisites
 - Complete all steps from `README.md`
+- Received permission from a testnet moderator or Axelar team member to run a validator
```

Comment:
don't want it to look like it's a "permissioned" system.
how about something softer ... 
"While the network is in development, you need to check in and receive an 'okay' from a testnet moderator or Axelar team prior to running a validator node"

Reply:
agree, sounds good